Build failure when targeting ARM64 due to missing declarations

```
../src/ws-egl.cpp: In destructor ‘virtual WS::ImplEGL::~ImplEGL()’:
../src/ws-egl.cpp:83:9: error: ‘close’ was not declared in this scope; did you mean ‘clone’?
   83 |         close(m_dmabuf.formatTable.fd);
      |         ^~~~~
      |         clone
../src/ws-egl.cpp: In member function ‘void WS::ImplEGL::initFormatTable()’:
../src/ws-egl.cpp:397:15: error: ‘ftruncate’ was not declared in this scope; did you mean ‘strncat’?
  397 |         ret = ftruncate(fd, size);
      |               ^~~~~~~~~
      |               strncat
../src/ws-egl.cpp:401:9: error: ‘close’ was not declared in this scope; did you mean ‘clone’?
  401 |         close(fd);
      |         ^~~~~
      |         clone
../src/ws-egl.cpp:407:9: error: ‘close’ was not declared in this scope; did you mean ‘clone’?
  407 |         close(fd);
      |         ^~~~~
      |         clone
```

--- a/src/ws-egl.cpp
+++ b/src/ws-egl.cpp
@@ -33,6 +33,7 @@
 #include <vector>
 #include <sys/mman.h>
 #include <sys/stat.h>
+#include <unistd.h>
 
 #ifndef EGL_WL_bind_wayland_display
 #define EGL_WAYLAND_BUFFER_WL 0x31D5

